web/docs/operators/sort.md

```diff
@@ -64,3 +68,9 @@ Arrange by field `foo` in descending order and put null values first:
 ```
 sort foo desc nulls-first
 ```
+
+Sort by the field `src_ip` and for matching values sort by `dest_ip`.
+
+```
+sort src_ip, dest_ip
```

Comment:
So if I want to specify the order I would write something like:
```
sort src_ip asc, src_bytes desc 
```
?

Reply:
Yes. Let me add an example.